README.markdown

```diff
@@ -6,3 +6,49 @@ A library to access the Twitter API using [Crystal][]
 [![Build Status](https://travis-ci.org/sferik/twitter-crystal.svg?branch=master)][travis]
 
 [travis]: https://travis-ci.org/sferik/twitter-crystal
+
+## Install
+
+The first thing you have to do is to create a `Projectfile`, in it, you can list your dependencies, as such:
+
+```crystal
+deps { github "sferik/twitter-crystal" }
+```
+
+In your terminal run:
+
+```bash
+crystal deps
+```
+
+This will get the latest code from this github repository and copy it to a `libs` directory. All that's left is to require it:
+
+```crystal
+require "./libs/twitter/twitter"
```

Comment:
You actually just need to do `require "twitter"` as "libs" is in the default crystal path.